src/SecurityUtilitiesPackageReference/SecurityUtilitiesApiUtilizationExample/SecurityUtilitiesApiUtilizationExample.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-	  <TargetFrameworks>net462;net6.0</TargetFrameworks>
+	  <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
Why are we dropping the check for `net462`? This will cause the `BuildAndTest.cmd` to crash, since it explicitly invokes the binaries in `net462`.